### PR TITLE
Fix #181496 the-decoder.com

### LINKED ISSUE #181496: the-decoder.com
### Issue URL (Annoyance)
 [https://the-decoder.com](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fthe-decoder.com)

### Screenshots
<details>
  <summary>Screenshot 1 </summary>

  Screenshot 1
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard for Safari v1.11.18
System version: | Sonoma
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances, <br/>AdGuard Cookie Notices, <br/>AdGuard Popups, <br/>AdGuard Other Annoyances<br/><br/><b>Other:</b><br/>Filter unblocking search ads and self-promotion<br/><br/><b>Language-specific:</b><br/>AdGuard Spanish/Portuguese

### PR DESCRIPTION
**Pull request by AlinaNaletova**
Fix https://github.com/AdguardTeam/AdguardFilters/issues/181496

### CODE REVIEW

File: AnnoyancesFilter/Cookies/sections/cookies_specific.txt [hunk 1]
```diff
@@ -273,6 +273,8 @@ consent-pref.trustarc.com#%#//scriptlet('trusted-click-element', 'div[aria-label
 !
 ! NOTE: Popular rules end ⬆️
 ! !SECTION: Cookies - Popular rules
+the-decoder.com##iframe#sp_message_iframe_929061
+the-decoder.com##div#sp_message_container_929061
```
Comment (Alex-302) [inline AnnoyancesFilter/Cookies/sections/cookies_specific.txt:277]: Wrong place.
Why not to block the cookie management system script?

Comment (Alex-302) [reply AnnoyancesFilter/Cookies/sections/cookies_specific.txt:277]: Use search in resources to find the necessary script.

Comment (AlinaNaletova) [reply AnnoyancesFilter/Cookies/sections/cookies_specific.txt:277]: should I change the place or change the rule?

Comment (Alex-302) [reply AnnoyancesFilter/Cookies/sections/cookies_specific.txt:277]: Both. Find the script which loads CMP.

Comment (zloyden) [inline AnnoyancesFilter/Cookies/sections/cookies_specific.txt:277]: Also remove these rules.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by AdamWr — APPROVED** [review]

File: AnnoyancesFilter/Cookies/sections/cookies_specific.txt (hunk 1 above)
Comment (AlinaNaletova) [reply AnnoyancesFilter/Cookies/sections/cookies_specific.txt]: done

**Review by zloyden — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**